### ISSUE TITLE
touchSession missing in v4 beta8

### ISSUE THREAD

**Issue by steelbrain** [auth0/nextjs-auth0#1829]
### Checklist

- [X] The issue can be reproduced in the [nextjs-auth0 sample app](https://github.com/auth0-samples/auth0-nextjs-samples/tree/main/Sample-01) (or N/A).
- [X] I have looked into the [Readme](https://github.com/auth0/nextjs-auth0#readme), [Examples](https://github.com/auth0/nextjs-auth0/blob/main/EXAMPLES.md), and [FAQ](https://github.com/auth0/nextjs-auth0/blob/main/FAQ.md) and have not found a suitable solution or answer.
- [X] I have looked into the [API documentation](https://auth0.github.io/nextjs-auth0/) and have not found a suitable solution or answer.
- [X] I have searched the [issues](https://github.com/auth0/nextjs-auth0/issues) and have not found a suitable solution or answer.
- [X] I have searched the [Auth0 Community](https://community.auth0.com) forums and have not found a suitable solution or answer.
- [X] I agree to the terms within the [Auth0 Code of Conduct](https://github.com/auth0/open-source-template/blob/master/CODE-OF-CONDUCT.md).

### Description

I am using the following snippet of code to make sure the session "rolls" over in my app router application.

```
import { touchSession } from '@auth0/nextjs-auth0/edge'
import { NextResponse, type NextRequest } from 'next/server'

export default async function middleware(req: NextRequest) {
  const res = NextResponse.next()
  await touchSession(req, res)

  return res
}
```

I just upgraded to Beta 14 so I can use Next.js v15 and I'm blocked on this

### Reproduction

Irrelevant

### nextjs-auth0 version

4.0.0-beta.8

### Next.js version

15.0.3

### Node.js version

v20.18.1

**Comment by sleitor** [comment]
Hello @steelbrain .
in v4.0.0 changed behavior little bit. Now touchSession() doing silently under getAccessToken()
regarding Documentation: https://github.com/auth0/nextjs-auth0/tree/v4?tab=readme-ov-file#getting-an-access-token

> If the access token has expired and a refresh token is available, it will automatically be refreshed and persisted.

starting from v4.0.0.beta8 getAccessToken() is Throwable. I.e. you will get error. You may look details there: https://github.com/auth0/nextjs-auth0/issues/1820#issuecomment-2499848210

**Comment by nandan-bhat** [comment]
Closing this ticket as this issue has been fixed `4.0.0-beta.9`
